src/utils/ledger.js

```diff
@@ -34,6 +34,38 @@ const formatPathData = (index) => {
   return buffer;
 };
 
+const intToByte = (number) => {
```

Comment:
I guess you have a method exactly like this in the lib at `hathorLib.helpersUtils.intToBytes(value, 1)`, where 1 is the number of bytes.

You can use both but using that method is more generic if we need to use more of int to bytes in the future.

Reply:
Started using it